lib/command_duration.bash

```diff
@@ -11,7 +11,10 @@ function _command_duration_pre_exec() {
 }
 
 function _dynamic_clock_icon {
-	local -i clock_hand=$(((${1:-${SECONDS}} % 12) + 90))
+	local clock_hand
+	# clock hand value is between 90 and 9b in hexadecimal.
+	# so between 144 and 155 in base 10.
+	clock_hand=$(printf '%x' $(((${1:-${SECONDS}} % 12) + 144)))
```

Comment:
* One too many `)`

* A lesser known feature of printf:
```
local clock_hand
# -v honors scope of variable if previously declared
printf -v clock_hand '%x' $(( (${1:-${SECONDS}} % 12) + 144 ))
```

Your test code used `{1..15}` but you _really_ want `{0..11}`

Using this, you find that `0` => "five-after the hour" instead of "midnight", which can be _slightly_ counter-intuitive.  i.e. Do you really want to show `five-after` if your calculation (seconds) is `0` ?

A quick way to shift the values so that `0` => "midnight" would be:
```
$(( ( (11 + ${1:-${SECONDS}}) % 12 ) + 144 ))
```

Additionally,

Does it really make sense to show hour-clock increments of 5 to represent single seconds?

i.e does it make sense that 1 second => 5 after and 2 seconds => 10 after?

Or do you want to round to the nearest clock value?  i.e. 1 second => midnight, 7 seconds => 5 after, 42 seconds => 40 after, etc?

And lastly,

Is this even meaningful once SECONDS > 60 ?

Reply:
> A lesser known feature of printf:

Good point, new commit will fix that.

>Your test code used `{1..15}` but you really want `{0..11}`

Yes, I was aiming for value > 12 to test the bugfix.
For {0..11}:

```bash
for second in {0..11}; do     
   printf -v clock_hand '%x' $(( (${1:-${SECONDS}} % 12) + 144 ))
   printf '%b' "\xf0\x9f\x95\x$clock_hand"
done

🕐🕑🕒🕓🕔🕕🕖🕗🕘🕙🕚🕛
```

> Additionally,
> Does it really make sense to show hour-clock increments of 5 to represent single seconds?

> Or do you want to round to the nearest clock value? i.e. 1 second => midnight, 7 seconds => 5 after, 42 seconds => 40 after, etc?

> And lastly,
> Is this even meaningful once SECONDS > 60 ?

¯\\_(ツ)_/¯

I was just fixing the bug.
A fixed icon or a dynamic one is ok for me, as long as the icon is displayed correctly.

If you need more info about the dynamic clock, @gaelicWizard [can give more info maybe](https://github.com/Bash-it/bash-it/commit/09e8c25b644315f3b3499776884d0c6f6c5cc863) ?